packages/expo-cli/lib/install-plugin.js

```diff
@@ -1,7 +1,7 @@
 const { spawn } = require('child_process')
 
 function resolveCommand (options) {
-  const command = ['install', '@bugsnag/plugin-expo-eas-sourcemaps']
+  const command = ['install', '@bugsnag/plugin-expo-eas-sourcemaps', '@bugsnag/source-maps']
```

Comment:
Is this needed if `@bugsnag/source-maps` is a direct dependency? (resolution here kinda depends on the answer to the other comment)